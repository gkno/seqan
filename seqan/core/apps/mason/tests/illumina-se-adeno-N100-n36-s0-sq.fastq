@read.000000000
TTCACCACGTGACTGGCAGCGACTCATCAACAACAA
+
IIHHGIHFHGIIGHHIIDIIEIIIIICHIIGIIIFH
@read.000000001
GGTTTATTGATTAACACGTAATTACAGGGGACGGGT
+
IHHHIIIIHGGFHGIDIIIGIHDIHFFIIGCHFBFF
@read.000000002
CGGGTGCCAATGGGGCGAGGCTCAGTATAAAGTCCA
+
IHHIIHFIIIFFHHIGAIIFEHIIIIFIGIB=IDDB
@read.000000003
CGCTGCCGCGTCAGCGCTGACGTAAATTACGTCATA
+
HIHHIHIEHIIIHIIEIICHGHIHIHFIEI@<AGII
@read.000000004
GTCGGCGCCTTCGTTATTGTCTGCCATTGGTGTGCC
+
IHHHHIFIIHIIFHDIIFIHIHIHIEGI=IGG?@I<
@read.000000005
GCGAGCAGGATCTCCATTTTGACCGCGAAATTTGAA
+
HHIIIHHHHIIGICIIFEIDDIIHIDAIFEHIIIGC
@read.000000006
TCCCGTCAATCACGGCGCACATGTTGGTGTTGGAGG
+
HIHHIIIIHEIIGIIIFGAHFEIHIIIIDIGGH?>B
@read.000000007
TCCACACTCACTTGTACTGTGGAGTATTGGGTGATG
+
IIHHHHHIHIFIHHIHDGIFIEIIIGCGCIIFEH=F
@read.000000008
CAGACTGGCGACTCAGAGTCAGTCCCCGATCCACAA
+
HHHIIGIHGIHHGHFDIIIFCIIIIIIBBII;IGEI
@read.000000009
CGGATGACAGGCGCGTCAGAATTGGGGTTCAGATCT
+
IIIHIHHFIIEHHIIHIIGEGBFFI@IIIGHCFG<I
@read.000000010
CTTTACCTTCAGCTACACCTTTGAGGAAGTGCCTTC
+
HIHHIIGHIIGIEIHIIFE>HIHIBIGIIIICIHI0
@read.000000011
CAGCTCCATGTAGCGCGCGGAGGTTTTTGACCGGAT
+
HHHGIHIGHIHIIIIIHCIIIFHEIIGIEHIIIEII
@read.000000012
CACATCTCCGGTCGCAGGGTCTGTGCTGCTGCTCTG
+
HIIIIIHGEDIGIIIFIIEIIICEIIIICC?FDEHI
@read.000000013
CGGCCAGGTTCAAACAGGCGCTTATATACTCCTCCA
+
IIIIHIHHGIIHIFFIIIIBCFIHGGIFCIEFIEGB
@read.000000014
GTACTCCGAGTCCGAGAAGGCTTGAACCGTGCTGGT
+
HHHHIHHIIIHIIGGGFFIFIIIGAIIIIDF=@IG7
@read.000000015
TCAGACGCGGAAGGAGCNCCGGTGGACTTTGCCGAC
+
IIIIHHFIIIIIHIIGI!HGDIIIBIIGI?IIEIII
@read.000000016
CACGGGGGTGGGGTCGATCTGGGCGGACGACTTGCG
+
HHHHIIIIGGHIIIEFIIAFIHCDA>IHICI=DIIE
@read.000000017
GCACCTTGATCACGATCTCGTAGAAGCCCGGCATGG
+
HHIHGHIIEHFIIIHII@IEDIFIIGIIIHHIII>C
@read.000000018
TCGCTCACTCGGCCATATATACCCTAAATGGCCACG
+
IIHIHGHHFHGFEHDFIFDFGIIAIIIIEI?HEBII
@read.000000019
GCAGGTGGTGCTGTTCCCGTCAATCACGGCGCACAT
+
IHHIIHIHHGHHIEEGIIIIEGGHIAEIG@CIBIFI
@read.000000020
TGTCGCAAAAGCACTCACGTGACAGCTAATACAGGA
+
HHIIHHIIIFIGHEGIEIAHFGIIFHIIFIIIIAII
@read.000000021
CAATGGCTTCAGGCGGTGGCGCACCAATGGCAGACA
+
IHHIIGIIIIIIIIFIGICGIGIDGIIIIIIIIIII
@read.000000022
TACGCGCACAGCCAGAGCCTGGACCGGCTGATGAAT
+
IHIIHHHIIHHIGIEIIDFICID?II?IIIIIIIII
@read.000000023
ATGCTCCCATAGCATGCACATCTCCGGTCGCNGGGT
+
IIIHIHIHFIIFEDFIIEHHIIIEIHGBDII!II?I
@read.000000024
AGTTGGGGATGTAGCACTCGTCCACCACCTAGTTCC
+
HIHHIHGGIGIIGIIIDHHDIIFFIBDIIDFIBAIH
@read.000000025
TTCGCGAGTGGTGGGACTTGAAACCTGGAGCCCCGA
+
HHHHIGIHHGGHGGFIIHIIDFICCIIHIIH@IIII
@read.000000026
AGTCAAGGAGGTCACGACGAATGATGGNGTCACAAC
+
HIHHIIIHFIGIGIHIIIIIIFIIIEF!DIIGIIBD
@read.000000027
GTAAAGTTGTTGCCCGTTCTCAGCATCTGAGAAGGG
+
HIHHIIIIFIFHIIIIDFFBHHIIFDGIIEDIIGIE
@read.000000028
GATGACCGGATAAGAGCGAGCCCAANGCGGGCCTGC
+
HHIHHFIHIHGIIAIHIBDIHIIII!EIEIIIIIII
@read.000000029
GTGACGAGAACATTTGTTTTGGTACCTGTCGGCAAA
+
IHIHIIHHHHIDHHIIIIIHIF<GHIIAIIIFCI>E
@read.000000030
CGGACGCAGCGTCCCTCGAGCACGACAAGGCCTACG
+
IHIGHGHIIGIIHIIFGCFFHGEIIDAGI?IEIIII
@read.000000031
GCAAGAAGATACGTCTTTTGGGGGCAACCTCGGGCG
+
HHHHIIHGGGEIFCHIIIEFIIHEGGFGIIIIII6F
@read.000000032
ACGAGGCCTGGTCCTCCTGGATCCACTGCTTCTCGG
+
IHHHGIIIHIFIEHIFGHEIIIFEGHGFAGEIEF;I
@read.000000033
CTCGGGCATCGGCAAGACAGGCCAGCAGCCCGCTAA
+
HHIIIHHIHIIIGFFHFEBICEBG:IHGHHEHIGDI
@read.000000034
GAAGGGAAAGTTCTCATTGGTCCAGTTGACGCAGCC
+
HIIIGGFIIIGIIGIFHIDFIIFIEHIHDIFHIFFI
@read.000000035
CCAAAACAAGGACTTGCTGTTTAGCCGTGGGTCTCC
+
HIHIHGHHIHIEIGIDIFIFEFHIHABFIIIIIIDI
@read.000000036
TCAACATTTGCTTCACGCACGGGACGAGAGACTGTT
+
HHIHIIHIHHGHIIFGIHIHFEIIIBIIIFDIIDII
@read.000000037
GATGACGGCCAAGGTCGTGGAGTCCGCCAAGGCCAT
+
IIIHHGHIIHIHFGHIHIGIIEGIIEH?GIFEFIDI
@read.000000038
ATGCTCCAGACGGCGGGTGAGTTAAATTTGAACATC
+
IHIHHHGIFIIIIIIEIBIHIAIAIFIIDIBIIIIC
@read.000000039
CCCGCGGTAGATGGTCTGCACCAGCTTGTCCCTAAT
+
IHHIHHIGIDFIIGDIGIGIIIGIGEBI@GIGII@G
@read.000000040
CCCTGGCACTGCTATGGCCTCACACAAAGACGACGA
+
IIHIGIHGGHIHIIIII?FIDGIIIIIIIII;>II>
@read.000000041
AGAAGGGCACGTGGGCGATGGCTTCCGCGATGTTGG
+
IHIHHIIIGGIDIIGIFIHDBEGCEIHIFIIEBIEI
@read.000000042
GATGCCCGAGCAGGAGTCTGGCTCTTGTGGAGACTG
+
HIIHIGIIIHIFHHI?IFECEII@CFIDIFIE?ICI
@read.000000043
TAGGGGTAACCCGATGACGTAAGTCTTTTATCGCGA
+
HHHIIIHFHIHIIIEIGIFIFDIIGGIF<II?FHAI
@read.000000044
CCCGAGATTGCTTGCTCGGCCTGCGATCTGGTCAAC
+
IHHIIIGIHIGIHFFHIHIIIGEIEB>IIHIIEHII
@read.000000045
AGCCNGGCATGGCTGCTGCTCGTTCAAATTTCGCGG
+
IHHI!IIHFHIHIEIIIIIEDGBEGIGEDIHFIIGI
@read.000000046
CGTAAATTACGTCATAGGGGAGTGGTCTTGTATTAG
+
IIHIGHGIIIIIGIHFHEIIAAICCG>HEGBDI@IH
@read.000000047
TCTGGTGGAGACCACGGGGGTCAAATCCATGGTGCT
+
HHIIHIHIGHIIHIIFIFIIHIIFI?DDGC@?BIDG
@read.000000048
GTGCCAATGGGGCGAGGCTCAGTATAAAGTCCATTG
+
HIHHIHIIHGIGHIGIIIHHI@IIGI9IGIIDIB?I
@read.000000049
CCCGCCGTCTGGAGCATGACTTTGGCAAGGTGACAA
+
IIHHIHGIGIHHIIIGHFIIBBIFIFIIGIIEIICI
@read.000000050
GATACTGTTTCCCTGCAAGACATGCGAGAGAATGAA
+
IHIFHHHHIHIIGIIHFIBECEDEIIFIIFH9>=I>
@read.000000051
TTGCAGACGCTCCTGAAACTCGGCGTCGGCGTGGTT
+
HIHHIIIGIFGHHIIIEHIGID=GIF;IGEIII>AA
@read.000000052
AACGCGCTGCTGCCGATAACAGGGTCCAGGTAGCCA
+
HHHHIHIIHIGIIIIGIBIIIIFBIIIIG?IFFID7
@read.000000053
GACAACAACAACAGCAATTTTACCTGGACTGGTGCT
+
HIHIIHGGHIGHEGIHIGAIGIIIIIIBCHFH>FI<
@read.000000054
CGCCCGACTACCTGGTAGGCCCCGCTCCGCCCGCGG
+
IHIIHHIIFIGGIIHIIGGIIH>EAIHIIIH?FFIH
@read.000000055
TCTGTGCCATTCATCATCTGCTGGGGCGGACTCCCG
+
IHHHHHGHIIGFIFGIIIGIFIII@I@I=FD@EIEI
@read.000000056
TCTTGCCCTCCTCCCACCAGATCACCATCTTGTCGA
+
IHIIHIIHIIIGHIHFIGHFIGHH=IFIHI=IIHIB
@read.000000057
CTGGGTGGCCGAGAAGGATGGGATCTGCCCCCGGAT
+
IIIIIIGHHHIIHFEHGIIDIIIIFEFI8;DIIIHG
@read.000000058
CCCCTGTAATTACGTGTTAATCAATAAACCTGTTGA
+
IIHIHIIGIFIGFFIIEIHIIFDIIEGIII@GII@C
@read.000000059
AATCCCCAATTGTTGTTGATGAGTCGCTGCCAGTCA
+
HHHHGIHGIIHFIHFGFIHDGIIIDFIIFIIAIIDG
@read.000000060
GGGCTGCCTCCCTCCGTTCCCGGCGGACGTGTTCAT
+
IHIIHHHGGIIFIIDGIFIIHBIFBHIIIIDGI:?G
@read.000000061
ACCTCCGAGAAGCAGTGGATCCAGGAGGACCAGGCC
+
HIIIIIIFGIIFIIAIICIBICDIEIIIIIHIBB;B
@read.000000062
CCCAATTCTGACGCGCCTGTCATCCGGTCAAAAACC
+
HIHHHHHIHGHFIHFI@IID=IIGIDIF@6EBGIIH
@read.000000063
CTCTTCAACATCCAAGTCAAGGAGGTCACGACGAAT
+
IHIHIHIIGFIIHIEFDIDHII@IIEAIIAI=DIII
@read.000000064
GGTCCACGTTGACCAGATCGCAGGCCGAGCAAGCAA
+
IIHHIIIIIIHIGIIDIHDIIHDHIHGFIIIDIHGH
@read.000000065
CGTCACAACCATCGCTAATAACCTTACCAGCACGGT
+
HHHIIHIIGIIIGFIHHEGDIHHDDI@BAHDIIIII
@read.000000066
GCAAGATCATGACGCTGACCAAATCCGCGCCCGACT
+
HHIIHIHIHIEGIIIIIIIIIEIHIEIHIAH@EIII
@read.000000067
ACCCAGCTCACAAACGAGTCAGAAATGCCCGGCAGG
+
HIHHHIIIIIIIHIIHIGHEIFBFIIBIIFIICIGI
@read.000000068
TGCCGACAGGTACCAAAACAAATGTTCTCGTCACGC
+
HIHHHIHIFGIHHIIIGIDIEIGIAI?GIFIIBIII
@read.000000069
GTCCACGTTGACCAGATCGCAGGCCAAGCAAGCAAT
+
HIHHHHIHIIIHIGFGIIIEIHIGACBIIHFAIEGI
@read.000000070
GACGACGAAGACAAGTTCTTTCCCATGAGCGGTGTC
+
HHHIGGHIIDGHIIHGGIIEHGDIBIIFGF?;IIII
@read.000000071
GAAATTGACTGCCACGGTCCCAAATCTTTCGGTGGC
+
HIHIHIHGHIGIIHFIHHIIIIIIIIIIIHEIIHII
@read.000000072
CCGGGGTCTGGTGCTTCCTGGCTACAAGTACCTAGG
+
IHIHHGGGHFIIIHICGIFICGEIIFIIGGII5GAI
@read.000000073
ATTCGTTTCAGTTGAACTTTGGTCTCCTGTCCTTCT
+
HIIIHGEHHGIGIIHGGIC?IIHGHIEEIIID?ADB
@read.000000074
GCGTCACAACCATCGCTAATAACCTTACCAGCACGG
+
IHHIGHIIGGFIFDHEHIHEFGBFHIBIEII@IHGG
@read.000000075
GCCGTTTGCGCTCGGCCAGGTTCAAACAGGCGCTTA
+
HHHIHHIIIGIFIGIIIIHIIHDG?IIIHAFIIGI@
@read.000000076
TGGACCGGCTGATGAATCCTCTCATCGACCAATACC
+
IIIGIHIHIHHHHIIIEBIHIIFIIIHHIIH@III>
@read.000000077
GCCCACTCCCTCTCTGCGCGCTCGCTCGCTCGGTGG
+
HHIHHIIIGHFGIAIIGIFCIIGI;IIDIHCIH@?;
@read.000000078
GCCCTTCTACGGCTGCGTCAACTGGACCAATGAGAA
+
HIHIHIHFGIIIIDIFIHIIGIFIHEIEHIHBII@@
@read.000000079
GACTCCACGACCTTGGCCGTCATCTTGCCCTCCTCC
+
IIHHIGGFHGHGIFGFIIIDGHIGGIIIBFEEEHAH
@read.000000080
TCCTTTTACTGCCTGGAATATTTCCCTTCTTAGATG
+
IHHIHIHHIFFFIIGEGFIDEEIIIIFIDI:FGFD<
@read.000000081
TCGCTCGCTCGGTGGGGCCTGCGGACCAAAGGTCCG
+
HHHHHGHHIHIHEIIBFFDIIHGIIHGGIDAIIIBC
@read.000000082
GTTACCTTACCCGTCCCCTGTAATTACGTGTTAATC
+
HIIHIHIGIFIEHDIFGGFIFGDIIFF@CI@IHFII
@read.000000083
TGGGACTTGAAACCATGGAGCCCCGAAGCCCAAAGC
+
IIIHGIHHGIIIIGDIIICIGGIEIIAIIB?IIGB:
@read.000000084
AGCAGCCATGCCGGGCTTCTACGAGATCGTGATCAA
+
HHIIIIIIIFIEGFIIIHDHIIFIGIICIGIIIAEI
@read.000000085
ACTTCGGCTACAGCACCCCCTGGGGGTATTTTGATT
+
HHHIIGHGIGIIGIHCGGIFHIGIIEIFIHIFH<GI
@read.000000086
TTGCGATTCCACATGGCTGGGCGACAGAGTCATCAC
+
IIHIGIGGIIFIFGIHEADIICDIHIACIIIIEIBD
@read.000000087
CTTCAAACTCTTCAACATCCAAGTCAAGGAGGTCAC
+
IHHIIIIIIGHIIIIIIIIIIGD@IHFFICFIAIII
@read.000000088
AGTATAAAGTCCATTGTTGTCCACAGTAAAATCAAC
+
IHHGGHIIIIIHIIGFIDIIIIHE>HIHICIIICHI
@read.000000089
GGGTGGCCGAGAAGGAATGGGATCTGCCCCCGGATT
+
HHIIHHIIGIGHFEGIICHEEIIIEEFIICBEH@CI
@read.000000090
TGAGGGCATTCGCGAGTGGTGGGACTTGAAACCTGG
+
IHIIHIIIHHHHEEIIIIIIIEIEFDDBFCAIIIIH
@read.000000091
TTTGAAGTTGAGTCTCTTGGGCCGGAATCCCCAATT
+
IHHIHHGFGIHGHIFIEHIIFIIHIIFBI@IHGIAG
@read.000000092
ATGGCCACGTGGTGTCGCAACAATGTCGCAAAAGCA
+
IHHHIIHIIHGEIFIIIIIG9HEIEGIFIIIFHHIG
@read.000000093
GACAAGTTCTTTCCCATGAGGCGGTGTCATGATTTT
+
HHIHIIFFIHIHGFCHDFFDIIIFIFIIIHIIIGCH
@read.000000094
TGTCCAGAGCGGCCTTGATCTGGGACCGCGAGTTGG
+
HHHHIHGIGIIIIGCIIIGIHHIIIIFDCIAH?IFI
@read.000000095
GCTAAAAAGAGACTCAATTTTGGTCAGACTGGCGAC
+
IHHIIHIHIHHIIHIHIIIFIIIFIHHEIIHHG=ID
@read.000000096
TGTACTGCACTTCGGGATTCCAGCGCTTGCTTGTTT
+
HHHHIIIIIFIFGG@IDICIHIIIIIBDICA;IIII
@read.000000097
CCGAGAAGCAGTGGATCCAGGAGGACCAGGCCTCGT
+
IHIIIHHHFGIGCIGEDHEAIFA@AIGIIICIEHII
@read.000000098
TCCTGCGAATCCTCCGGCGGAGGTTTCAGCTACAAA
+
HIIIIIHIIHIGFHHIIIIIIE?FDIHFGIGHIG:I
@read.000000099
CCGGAGCTCCTTCCGCGTCTGACGTCGATGGATCCG
+
IIIIIHHFHIHHEGIIHIDIBBFAIHHIGCIIIIHF
