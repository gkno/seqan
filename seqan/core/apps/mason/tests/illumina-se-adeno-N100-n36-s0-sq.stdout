  Building haplotype...
    contig # 1/1
  Finished haplotype creation in 7.032e-05s
